src/components/InventoryTable/hooks/useInventoryExport.js

```diff
@@ -0,0 +1,42 @@
+import { useCallback } from 'react';
+import { useAxiosWithPlatformInterceptors } from '@redhat-cloud-services/frontend-components-utilities/interceptors';
+
+const EXPORT_SERVICE_PATH = '/api/export/v1/exports';
+const EXPORT_SERVICE_APPLICATON = 'urn:redhat:application:inventory';
```

Comment:
For my knowledge, what does `urn:redhat:application:inventory` mean ?

Reply:
That is the key of the export service that the inventory will look at to receive export requests on the backen.